<launch>
	
    <!-- Task ('basic' - standard trajectory from shelf to table, 'tilted' -
    same as basic, but the cup in the end effector is tilted) -->
	<arg name="task" default="basic"/> 

	<!-- List of string features the robot contains
    ('efficiency', 'table', 'coffee', 'human', 'laptop', 'origin') -->
	<arg name="feat_list" default="table"/>

	<!-- List of feature weights the robot is initialized with. -->
	<arg name="feat_weight" default="0"/>

	<!-- Velocity-based with trajopt planning PID -->
	<node name="pid_control" pkg="beta_adaptive_pHRI" type="pid_control.py"
    $(arg task) $(arg feat_list) $(arg feat_weight)" output="screen"/>

</launch>
